Components/ListSearchResults/Partials/Header/_style.scss

```diff
@@ -1,59 +1,83 @@
+$search-form-spacing: 6px;
+
 .header {
   &-title {
```

Comment:
Could you please remove the -title element and instead implement a preContent approach like used in other recently developed components (e.g. BlockCountUp)?